third_party/move/move-vm/integration-tests/src/tests/regression_tests.rs

```diff
@@ -110,7 +110,7 @@ fn script_large_ty() {
     let vm_config = VMConfig {
         verifier_config,
         paranoid_type_checks: true,
-        ..Default::default()
+        ..VMConfig::default_for_test()
```

Comment:
Looks like some subset of (integration) tests set paranoid type checks true (like this one), but not all.

I am assuming the intention is `default_for_test` is have paranoid type checks true (does not have to be in this PR). I can use the same mechanism to set reference safety checks true.